doc/role-icinga2/objects.md

```diff
@@ -7,15 +7,23 @@ generate configuration files with objects included.
 
 This variable consists of Icinga 2 object attributes and attributes referring to the file created in the process.
 
-> **_NOTE:_** The second level of the dictionary defines on which host the configuration is created. All objects in the example below, will be gathered and deployed on the host.: `host.example.org`.
-In addition this variable can be logically defined at the **host_vars/agent** and are still deployed on the master **host.example.org**
+> **_NOTE:_** The second level of the dictionary defines on which host the configuration is created. All objects in the example below, will be gathered and deployed on the host.: `host.example.org`.  
+In addition this variable can be logically defined at the **host_vars/agent** and are still deployed on the master **host.example.org**  
+The second level can **only** be used in **hostvars**!
 
 The `file` key will be used to control in which directory structure the object will be placed.
 In addition the `order` key will define the order of the objects in the destination file.
 The default for `order` is set to **10**, so everything below that number will be in front of the file, all objects with the order above 10 will be placed later in the file.
 
 The `type` will be the original Icinga 2 object types, a list of all can be found in the documentation. [Icinga 2 Monitoring Objects](https://icinga.com/docs/icinga-2/latest/doc/09-object-types/#monitoring-objects)
 
+### Icinga2 Objects in Hostvars
+
+When defining `icinga2_objects` as a host specific variable (hostvars/groupvars) you can define the variable as a dictionary. Each dictionary key represents the host on which the key's value will be deployed as configuration.  
+Alternatively you can define `icinga2_objects` as a list which results in the configuration being deployed on just the host for which the variable is defined.
```

Comment:
Can you add a example how the "Icinga2_objects" variable as list should look like?

Reply:
Done right here
[https://github.com/Icinga/ansible-collection-icinga/blob/48805b98dcdd402005044b3f83d0d1eeae87d626/doc/role-icinga2/objects.md?plain=1#L43-L57](https://github.com/Icinga/ansible-collection-icinga/blob/48805b98dcdd402005044b3f83d0d1eeae87d626/doc/role-icinga2/objects.md?plain=1#L43-L57)